authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -0,0 +1,234 @@
+/*
+Copyright The Guard Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package rbac
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	authzv1 "k8s.io/api/authorization/v1"
+)
+
+type SubjectInfoAttributes struct {
+	ObjectId              string   `json:"ObjectId"`
```

Comment:
kinda weird to see json property starts with upper case

Reply:
Its as per checkaccess spec.